### DISCUSSION TITLE
"Section 3 Control Plane Configuration" in CIS Kubernetes Benchmark v1.8.0 is not applicable for K3s hardening ?

### DISCUSSION THREAD

**Discussion by pentester0006** [k3s-io/k3s#10590] [General]
### Why is "Section 3 Control Plane Configuration" in CIS Kubernetes Benchmark v1.8.0 is not applicable for K3s hardening https://docs.k3s.io/security/self-assessment-1.8

**Comment by caroline-suse-rancher** [comment]
Hey, this is more of a question as opposed to a bug or enhancement/feature request, so I'm going to convert this to a discussion.
